tests/coverage/clover.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I don't think we want to be committing these tests into this repo, we should probably make a secondary repo that we use to test whether module changes work as expected.

Reply:
I see your point and agree.
I removed it from the library and create a new repo with the tests.

https://github.com/samuelpampolini/cypress-cucumber-attach-screenshots-to-failed-steps-tests